torch/_inductor/compile_fx.py

```diff
@@ -120,6 +120,18 @@ def complex_memory_overlap(t: torch.Tensor) -> bool:
     return False
 
 
+def get_static_input_idxs(num_fixed):
+    # If we are inlining NNModules, we treat all torch.nn.Parameters as static for the purposes
+    # of cudagraphs. Rather than copying these into cudagraph-owned memory
+    # like we do for normal inputs on each run, we will re-record a cudagraph if these
+    # parameter locations change.
+    context = torch._guards.TracingContext.try_get()
+    if not context or not context.fw_metadata:
+        return []
```

Comment:
nit.. should we just return list(range(num_fixed)) here ?  dont know if that will ever happen